plugins/outputs/elasticsearch/elasticsearch.go

```diff
@@ -0,0 +1,271 @@
+package elasticsearch
+
+import (
+	"fmt"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"gopkg.in/olivere/elastic.v3"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type Elasticsearch struct {
+	ServerHost       string
+	IndexName        string
+	EnableSniffer    bool
+	HealthCheck      bool
+	NumberOfShards   int
+	NumberOfReplicas int
+	Client           *elastic.Client
+	Version          string
+}
+
+var sampleConfig = `
+  server_host = "http://10.10.10.10:9200" # required.
+  index_name = "test" # required.
```

Comment:
most of these config options seem to be elastic config options, can you provide links to documentation?